kube_scheduler/metadata.csv

```diff
@@ -0,0 +1,38 @@
+kube_scheduler.apiserver.audit.event.count,count,,,,"Number of audit events generated and sent to the audit backend",0,kube_controller_manager,apiserver.audit.event.count
+kube_scheduler.apiserver.audit.requests_rejected.count,count,,,,"Number of apiserver requests rejected due to an error in audit logging backend",0,kube_controller_manager,apiserver.audit.requests_rejected.count
+kube_scheduler.apiserver.client_certificate_expiration.count,gauge,,,,"Number of the remaining lifetime on the certificate used to authenticate a request",0,kube_scheduler,apiserver.client_certificate_expiration.count
+kube_scheduler.apiserver.client_certificate_expiration.sum,gauge,,,,"Sum of the remaining lifetime on the certificate used to authenticate a request",0,kube_scheduler,apiserver.client_certificate_expiration.sum
+kube_scheduler.apiserver.storage.data_key_generation_failures.count,gauge,,,,"Number of failed data encryption key(DEK) generation operations",0,kube_scheduler,apiserver.storage.data_key_generation_failures.count
+kube_scheduler.apiserver.storage.envelope_transformation_cache_misses.count,gauge,,,,"Number of cache misses while accessing key decryption key(KEK)",0,kube_scheduler,apiserver.storage.envelope_transformation_cache_misses.count
+kube_scheduler.binding_duration.count,gauge,,,,"Number of latency in seconds",0,kube_scheduler,binding_duration.count
+kube_scheduler.binding_duration.sum,gauge,,,,"Total binding latency in seconds",0,kube_scheduler,binding_duration.sum
+kube_scheduler.cache.lookups.count,count,,,,"Number of equivalence cache lookups, by whether or not a cache entry was found",0,kube_scheduler,cache.lookups.count
+kube_scheduler.client.http.requests,count,,,,"Number of HTTP requests, partitioned by status code, method, and host",0,kube_scheduler,client.http.requests
+kube_scheduler.client.http.requests_duration.count,gauge,,,,"Number of client requests. Broken down by verb and URL",0,kube_scheduler,http.requests_duration.count
+kube_scheduler.client.http.requests_duration.sum,gauge,,,second,"Total latency. Broken down by verb and URL",0,kube_scheduler,http.requests_duration.sum
+kube_scheduler.gc_duration_seconds.count,gauge,,,,"Number of the GC invocation",0,kube_controller_manager,gc_duration_seconds
+kube_scheduler.gc_duration_seconds.quantile,gauge,,,second,"GC invocation durations quantiles",0,kube_controller_manager,goroutines
+kube_scheduler.gc_duration_seconds.sum,gauge,,,second,"GC invocation durations sum",0,kube_controller_manager,goroutines
+kube_scheduler.goroutines,gauge,,,,"Number of goroutines that currently exist",0,kube_controller_manager,goroutines
+kube_scheduler.max_fds,gauge,,,,Maximum allowed open file descriptors,0,kube_scheduler,max_fds
+kube_scheduler.open_fds,gauge,,,,Number of open file descriptors,0,kube_scheduler,open_fds
+kube_scheduler.pod_preemption_victims.count,counter,,,,"Number of selected preemption victims",0,kube_scheduler,pod_preemption_victims.count
```

Comment:
From reading the code, the metric seems to be `Number of selected pods during the latest preemption round`. Is it?

Reply:
the counter is incremented here: https://github.com/kubernetes/kubernetes/blob/057ad6d4ce3a1fc04789dc4e7c877974576ac34c/pkg/scheduler/scheduler.go#L470
And I think it is the correct description.

Reply:
Agree